Stardaze/OutputFormatter.swift

```diff
@@ -1,12 +1,18 @@
 //
-//  PrettyPrintedStringFormatter.swift
+//  OutputFormatter.swift
 //  Stardaze
 //
 //  Created by William Wilson on 3/7/17.
 //  Copyright © 2017 LeTote. All rights reserved.
 //
 
-internal struct PrettyPrintedStringFormatter: Visitor {
+internal struct OutputFormatter: Visitor {
+    private var outputOption: OutputOption
+
+    internal init(outputOption: OutputOption) {
+        self.outputOption = outputOption
```

Comment:
I think a more appropriate name for `OutputOption` would be `OutputFormat`. The word "option" is a bit too generic, whereas "format" implies what it is that you are specifying exactly.